kratos/geometries/nurbs_curve_tessellation.h

```diff
@@ -0,0 +1,161 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Thomas Oberbichler
+//
+//  Ported from the ANurbs library (https://github.com/oberbichler/ANurbs)
+//
+
+#if !defined(KRATOS_NURBS_CURVE_TESSELLATION_H_INCLUDED )
+#define  KRATOS_NURBS_CURVE_TESSELLATION_H_INCLUDED
+
+#include "nurbs_curve_geometry.h"
+
+#include "includes/ublas_interface.h"
+#include "containers/array_1d.h"
+
+namespace Kratos {
+
+template <typename TCurve>
+class NurbsCurveTessellation
+{
+public:     // types
+    using Vector = array_1d<double, TCurve::GetDimension()>;
+    using ParameterPoint = std::pair<double, Vector>;
+
+private:    // static methods
+    static double Cross(const array_1d<double, 2>& VectorU,
```

Comment:
This is not cross product, this is the determinant isn't it?

Reply:
It's the cross product of the 2D vectors in 3D space. Of course the cross product is the same as the determinant but in a geometric context the name 'cross product' makes more sense to me.

Reply:
But then it would be the norm of the cross product